shared/src/main/scala/coop/rchain/shared/package.scala

```diff
@@ -18,3 +18,4 @@ trait AllSyntaxShared
     with KeyValueStoreManagerSyntax
     with MonixableSyntax
     with Fs2StreamSyntax
+    with catscontrib.ToBooleanF
```

Comment:
Just curious.
This is not related to byteString, right?

Reply:
I'm ok for ToBooleanF to also be included in this PR. It needs to be renamed.